src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -70,11 +70,22 @@ public Claim getClaimByExternalId(final String externalId) {
             .orElseThrow(() -> new NotFoundException("Claim not found by external id " + externalId));
     }
 
+    public Optional<Claim> getClaimByReference(final String reference, final String authorisation) {
+        final String submitterId = userService.getUserDetails(authorisation).getId();
+        return claimRepository
+            .getByClaimReferenceAndSubmitter(reference, submitterId);
+    }
+
     public Optional<Claim> getClaimByReference(final String reference) {
```

Comment:
do we still need this method?? I agree we need to update T1 frontend?

Reply:
Yes, it is used by `isDefendantLinked`